crates/ethereum/evm/src/execute.rs

```diff
@@ -197,7 +202,10 @@ where
             .into())
         }
 
-        Ok((receipts, cumulative_gas_used))
+        let withdrawal_requests =
+            post_block_withdrawal_requests(&self.chain_spec, block.timestamp, evm)?;
+
+        Ok((receipts, cumulative_gas_used, withdrawal_requests))
```

Comment:
TODO: we should convert this to a struct later

Reply:
converted now